tests/test_fluxFitBoundedField.py

```diff
@@ -142,21 +146,18 @@ def setUp(self):
             self.ffp[ccd] = lsst.meas.mosaic.FluxFitParams(fcrMetadata)
             wcsFilename = os.path.join(
                 DATA_DIR,
-                "%d/wcs-%07d-%03d.fits" % (self.tract, self.visit, ccd)
+                "%d/jointcal_wcs-%07d-%03d.fits" % (self.tract, self.visit, ccd)
```

Comment:
Yes, it was renamed: see `mosaicTask.py::646`.

The goal of the rename was to make it more obvious what the object was. jointcal and meas_mosaic have identical output so that the input to calibration/validation can be swapped between them.